_studio/shared/mfx_trace/src/mfx_trace_utils_linux.cpp

```diff
@@ -50,46 +46,49 @@ FILE* mfx_trace_open_conf_file(const char* name)
 
     if (home)
     {
-        snprintf_s_ss(file_name, MAX_PATH-1, "%s/.%s", home, name);
-        file = fopen(file_name, "r");
+        ss << home << "/." << name;
+        file = fopen(ss.str().c_str(), "r");
     }
     if (!file)
     {
-        snprintf_s_ss(file_name, MAX_PATH-1, "%s/%s", MFX_TRACE_CONFIG_PATH, name);
-        file = fopen(file_name, "r");
+        ss << MFX_TRACE_CONFIG_PATH << "/" << name;
+        file = fopen(ss.str().c_str(), "r");
     }
     return file;
 }
 
 /*------------------------------------------------------------------------------*/
 
-mfxTraceU32 mfx_trace_get_value_pos(FILE* file,
-                                    const char* pName,
-                                    char* line, mfxTraceU32 line_size,
-                                    char** value_pos)
+inline std::string& trim_string(std::string&& s)
+{
+    // Remove leading spaces, tabulation, = character
+    s.erase(0, s.find_first_not_of(" =\t"));
+    // Remove trailing spaces, tabulation, newline symbol, spaces
+    s.erase(s.find_last_not_of(" \t\n\r\f\v") + 1);
```

Comment:
What will happen if find will return npos?

Reply:
It will throw out_range exception. Will fix it